src/containers/EmptyGraphLayoutContainer.tsx

```diff
@@ -75,17 +86,34 @@ export class EmptyGraphLayout extends React.Component<EmptyGraphLayoutProps, Emp
     const isGraphEmpty = !this.props.elements || !this.props.elements.nodes || this.props.elements.nodes.length < 1;
 
     if (isGraphEmpty) {
+      console.log(this.props.isDisplayingUnusedNodes);
       return (
         <EmptyState className={emptyStateStyle}>
           <EmptyStateTitle>Empty Graph</EmptyStateTitle>
           <EmptyStateInfo>
             There is currently no graph available for namespace <b>{this.props.namespace}</b>. This could either mean
-            there are no service mesh available in this namespace or that nothing has accessed the service mesh. Please
-            try accessing something in the service mesh and click 'Refresh'.
+            there are no service mesh available in this namespace or that nothing has accessed the service mesh.
+            {this.props.isDisplayingUnusedNodes && (
+              <>
+                {' '}
+                You are currently displaying 'Unused nodes', please try accessing something in the service mesh and
+                click 'Refresh'.
+              </>
+            )}
+            {!this.props.isDisplayingUnusedNodes && (
+              <>
+                {' '}
+                You can turn 'Unused nodes' to display nodes that don't show yet any interaction in the service mesh.
```

Comment:
Minimal: "You can enable 'Unused nodes'..."

Suggested: "You can enable 'Unused nodes' to display service mesh nodes that have yet to see any request traffic."